src/bin/errors.ts

```diff
@@ -88,9 +72,6 @@ export {
   ErrorCLIPasswordFileRead,
   ErrorCLIRecoveryCodeFileRead,
   ErrorCLIFileRead,
-  ErrorSecretPathFormat,
-  ErrorVaultNameAmbiguous,
-  ErrorSecretsUndefined,
-  ErrorNodeFindFailed,
-  ErrorNodePingFailed,
+  ErrorCLINodeFindFailed,
+  ErrorCLINodePingFailed,
```

Comment:
Shouldn't be necessary.

Reply:
Keeping these as the commands still fail with these errors when a timeout is set.

Reply:
But but why are these not domain errors?